js/core/memorized_callbacks.js

```diff
@@ -1,27 +1,23 @@
-const each = require('../core/utils/iterator').each;
-const Callbacks = require('./utils/callbacks');
+import { each } from '../core/utils/iterator';
+import Callbacks from './utils/callbacks';
 
-const MemorizedCallbacks = function() {
+export default class MemorizedCallbacks {
+    constructor() {
+        this.memory = [];
+        this.callbacks = Callbacks();
```

Comment:
Should we rewrite `Callbacks` to `class`?

Reply:
I think yes, but in other PR.